profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry_mock/tmgmt_poetry_mock.behat.inc

```diff
@@ -228,25 +228,19 @@ class TMGMTPoetryMockSubContext extends RawDrupalContext implements DrupalSubCon
    */
   public function assertTranslationRequestDocumentIsValidXhtml() {
 
-    if (!class_exists('tidy')) {
-      throw new Exception('You need Tidy extension to test if XHTML is valid.');
-    }
+    $msg = 'You need Tidy extension to test if XHTML is valid.';
+    assert(class_exists('tidy'), equals(TRUE), $msg);
```

Comment:
What is the problem of having an extra "use function" entry at the top ?
The overhead added by an extra "use function" is not that huge.
I really think it's more important to have easier tests to understand than avoiding the use of an extra "use function" line.